hypha/static_src/src/app/webpack.prod.config.js

```diff
@@ -13,6 +13,8 @@ prodConfig.plugins = prodConfig.plugins.concat([
         NODE_ENV: 'production',
         API_BASE_URL: null ,
     }),
+    // Used in local development, where React app is precompiled and webpack server is not actively running.
+    new BundleTracker({filename: './hypha/static_compiled/app/webpack-stats.json'}),
```

Comment:
Interesting. I was thinking of re-using the prod command.

I'll give this a try.

Reply:
@vimal1083 The `app` task produces a `webpack-stats.json` file which is only compatible with the webpack server.

It contains the `publicPath` property which makes webpack-loader load it from webpack server (which is not running). The scripts then fail to load because of this.

An alternative for this is to define a webpack config similar to production, but which only used in the `app` task.

How should I proceed?

Reply:
Try to parameterize the webpack config, so that we can have same file for both environments. If that is not possible we can have separate webpack config file for dev

Reply:
I ended up going with a separate config, parameterizing the config was making it unnecessarily complicated.